web/private/scripts/slack_deploy_notification.php

```diff
@@ -162,12 +162,11 @@ function _get_secrets($requiredKeys, $defaults)
 /**
  * Send a notification to slack
  */
-function _slack_notification($slack_url, $channel, $username, $text, /*$attachment,*/ $alwaysShowText = true)
+function _slack_notification($slack_url, $username, $text, /*$attachment,*/ $alwaysShowText = true)
 {
   // $attachment['fallback'] = $text;
   $post = array(
     'username' => $username,
-    'channel' => $channel,
```

Comment:
Why didn't you remove username here as well? If you're not going to pass in that parameter you'll need to remove it here too. Are you sure this function with work without the username and channel values?

Reply:
Okay... trying again with a focus on only touching the channel.